### PR TITLE
adding create_inbound_shipment_plan

### PR DESCRIPTION
**Pull request by vahakmatavosian**
adding create_inbound_shipment_plan

### CODE REVIEW

File: lib/peddler-ratelimited/create_inbound_shipment_plan.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    end
+
+    def self.act(args)
+      binding.pry
+      result = call_feed(args)
+      binding.pry
+
+      process_feeds_list(result, args[:processor])
+    rescue Exception => e
+      binding.pry
```
Comment (kitwalker12) [inline lib/peddler-ratelimited/create_inbound_shipment_plan.rb]: forgot to remove this and the ones above

File: lib/peddler-ratelimited/create_inbound_shipment_plan.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      log_error('create_inbound_shipment_plan', result, e, args)
+    end
+
+    def self.call_feed(args)
+      AmazonMWS.instance.
+        inbound_fulfillment.
+        create_inbound_shipment_plan(args[:ship_from_address],
```
Comment (kitwalker12) [inline lib/peddler-ratelimited/create_inbound_shipment_plan.rb:29]: are these args objects/arrays?

Comment (vahakmatavosian) [reply lib/peddler-ratelimited/create_inbound_shipment_plan.rb:29]: yes, array of hashes but it looks like Resque is ok with them.

File: peddler-ratelimited.gemspec [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |gem|
   gem.name        = 'peddler-ratelimited'
-  gem.version     = '1.0.3'
+  gem.version     = '1.0.4'
   gem.date        = '2017-08-10'
   gem.summary     = "RateLimited Peddler!"
   gem.description = "Rate limitting Peddler"
```
Comment (kitwalker12) [inline peddler-ratelimited.gemspec:6]: add `gem.add_dependency 'peddler', '~> 1.6.1'`

**Review by kitwalker12 — APPROVED** [review]

**Pull request merged**